chains/ethereum/writer_methods.go

```diff
@@ -26,36 +26,41 @@ var NonceRetryInterval = time.Second * 2
 
 const TxRetryLimit = 10
 
+// constructErc20ProposalData returns the bytes to construct a proposal suitable for Erc20
 func constructErc20ProposalData(amount []byte, resourceId msg.ResourceId, recipient []byte) []byte {
 	var data []byte
-	data = append(data, resourceId[:]...)                   // resourceId
-	data = append(data, common.LeftPadBytes(amount, 32)...) // amount
+	data = append(data, resourceId[:]...)                   // resourceId (bytes32)
```

Comment:
Just a small nit here: could you tab these comments out such that they are all aligned?

Reply:
They seem to be aligned on my codebase, but it spacing is removed when I push it. Is there a formatter being run?
![format](https://user-images.githubusercontent.com/29740020/81101649-7380ed80-8edc-11ea-9ef3-ea9a7fdf7148.png)

Reply:
Possibly `go fmt`, which is required by our linter. VSC may apply automatically when you commit.